Gemfile.lock

```diff
@@ -0,0 +1,239 @@
+GEM
```

Comment:
@jmanero-r7 are we sure we want to commit Gemfile.lock? Standard practice is that you commit it for application code, but not for library code. [This blog post](http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) summarises the reasons why.